modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/persistence/db/LongDestroyDurableBackgroundTaskTest.java

```diff
@@ -461,6 +471,41 @@ private IgniteEx prepareAndPopulateCluster(int nodeCnt, boolean multicolumn) thr
         return ignite;
     }
 
+    /**
+     * Test case when cluster deactivation happens with no-persistence cache. Index tree deletion task should not be
+     * started after stopping cache.
+     *
+     * @throws Exception If failed.
+     */
+    @Test
+    public void testClusterDeactivationShouldPassWithoutErrors() throws Exception {
```

Comment:
Let's check messages that tasks weren't started/finished after deactivation. Also, it would be great to check that all nodes still alive after deactivation: add failure handler to config and try to get node instance after deactivation.